db/seeds.rb

```diff
@@ -16,6 +16,55 @@ def rugged_repo
   end
 end
 
+tags = [
+  [
+    "Compiles to:Binary",
+    "Compiles to:Bytecode",
+    "Compiles to:JavaScript",
+  ],
+  [
+    "Runtime:Android",
+    "Runtime:Browser",
+    "Runtime:BEAM (Erlang)",
+    "Runtime:Common Language Runtime (.NET)",
+    "Runtime:iOS",
+    "Runtime:JVM (Java)",
+    "Runtime:V8 (NodeJS)",
+  ],
+  [
+    "Paradigm:Declarative",
+    "Paradigm:Functional",
+    "Paradigm:Imperative",
+    "Paradigm:Logic",
+    "Paradigm:Object-oriented",
+    "Paradigm:Procedural",
+  ],
+  [
+    "Typing:Static",
+    "Typing:Dynamic",
+  ],
+  [
+    "Family:C-like",
+    "Family:Lisp",
+    "Family:ML",
+    "Family:Java",
+    "Family:JavaScript",
+    "Family:SASL",
+    "Family:sh",
+  ],
+  [
+    "Domain:Cross-platform",
+    "Domain:Embedded systems",
+    "Domain:Games",
+    "Domain:GUIs",
+    "Domain:Logic",
+    "Domain:Maths",
+    "Domain:Mobile",
+    "Domain:Scripting",
+    "Domain:Web development",
+  ]
+]
```

Comment:
I've not 100% decided how to model this, so I'm leaving this as stub data for now. We need to think about it from the OSS POV, which I'll want @ErikSchierboom's expertise on. This is ok for now though I think.

Reply:
Yeah let's postpone this.